src/main/java/tetris/PlayerSkeleton.java

```diff
@@ -32,13 +32,15 @@ public static void main(String[] args) {
     State s = new State();
     new TFrame(s);
     PlayerSkeleton p = new PlayerSkeleton();
+    ArrayList<Feature> features = new ArrayList<Feature>();
+    features.add(new RowsClearedFeature());
+    features.add(new TotalHeightFeature());
+    features.add(new HoleFeature());
+    features.add(new UnevenFeature());
+
+    Heuristic h = new Heuristic(features);

```

Comment:
this should still be a player property, so the function signature of pickMove doesn't have to include heuristic.

Reply:
realised can't cos playerskeleton class is static, so it's fine